src/frontend/components/UI/ImportGameFolder/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import TextInputWithIconField from '../TextInputWithIconField'
+import { useTranslation } from 'react-i18next'
+import { faBackspace, faFolderOpen } from '@fortawesome/free-solid-svg-icons'
+import { Button, LoadingSpinner } from '@hyperplay/ui'
+import { useMutation } from '@tanstack/react-query'
+import styles from './index.module.scss'
+
+export function ImportGameFolder() {
+  const { t } = useTranslation()
+  const [gameFolderToImport, setGameFolderToImport] = useState('')
+
+  const importGameFolderMutation = useMutation({
+    mutationFn: async (gameFolder: string) => {
+      const result = await window.api.importGameFolder(gameFolder)
+      return result
```

Comment:
Great point. I added a success message now that importGameFolder returns. 

Since this is in advanced settings though, I'd like to handle further UI/UX improvements as needed on separate branches in the future. We would likely need some design input as well and I'm not sure if it's high priority at the moment.